code/game/objects/items/storage/fancy.dm

```diff
@@ -23,10 +27,10 @@
 	var/spawn_type
 	/// How many of the things to fill this storage with.
 	var/spawn_count = 0
-	/// Whether the container is open or not
-	var/is_open = FALSE
-	/// What this container folds up into when it's empty.
-	var/obj/fold_result = /obj/item/stack/sheet/cardboard
+	/// Whether the container is open, always open, or closed
+	var/is_open = CONTAINER_CLOSED
```

Comment:
this could also be something like `open_status` since it's more than just a standard boolean